content/docs/tasks/traffic-management/ingress/index.md

```diff
@@ -279,6 +279,38 @@ available for edge services.
 In the preceding steps, you created a service inside the service mesh
 and exposed an HTTP endpoint of the service to external traffic.
 
+## Troubleshooting
+
+1.  Inspect the values of the `INGRESS_HOST` and `INGRESS_PORT` environment variables. Make sure
+they have valid values, according to the output of the following commands:
+
+    {{< text bash >}}
+    $ kubectl get svc -n istio-system
+    $ echo INGRESS_HOST=$INGRESS_HOST, INGRESS_PORT=$INGRESS_PORT
+    {{< /text >}}
+
+1.  Check that you have no other Istio ingress gateways defined on the same port:
+
+    {{< text bash >}}
+    $ kubectl get gateway --all-namespaces
+    {{< /text >}}
+
+1.  Check that you have no other Kubernetes Ingress resources defined on the same IP and port:
+
+    {{< text bash >}}
+    $ kubectl get ingress --all-namespaces
+    {{< /text >}}
+
+1.  Check the logs of the ingress gateway and verify that your request appears in the logs. If your request is not there,
```

Comment:
This step won't show the information in 1.1 because `accessLog` is now disabled by default